channels/tr.m3u

```diff
@@ -132,6 +134,8 @@ https://tv-trtavaz.live.trt.com.tr/master.m3u8
 https://tv-trtcocuk.live.trt.com.tr/master.m3u8
 #EXTINF:-1 tvg-id="TRT Haber TR" tvg-name="TRT Haber TR" tvg-language="Turkish" tvg-logo="https://i.ibb.co/3pKN9gx/BYsl7Nc.jpg" group-title="News",TRT Haber
 https://tv-trthaber.live.trt.com.tr/master.m3u8
+#EXTINF:-1 tvg-id="TRT Belgesel" tvg-name="TRT Belgesel" tvg-language="Turkish" tvg-logo="https://i.ibb.co/h1FcR9W/trt-belg.png" group-title="",TRT Belgesel
+https://tv-trtbelgesel.live.trt.com.tr/master.m3u8
```

Comment:
This link no longer works. Tested in VLC, Elmedia Video Player and streamtest.in

Reply:
actually this link is the legal one however there is geo restriction I think, i'll remove it untill I find something better